src/css/style.css

```diff
@@ -3,3 +3,94 @@
   margin: 0;
   padding: 0;
 }
+
+.container{
+  background-color: #DBC2F5;
+  display: flex;
+  max-height: 100vh;
+  text-align: left;
+  font-size: 20px;
+  font-family: 'DM Sans', sans-serif;
+  
+}
+
+
+.sub-container {
+  margin-left: 05%;
+  margin-top: 15%;
+  max-width: 60%;
+  
+  margin-right: 20%;
+
+   
+   
+  
+
+
+}
+
+
+body{
+  background-color: #DBC2F5;
+}
+
+.pic img {
+  padding-right: 20%;
+  margin-top: 15%; 
+  margin-left: 20%;
+  margin-bottom: 20%;
+  
+  max-width: 500px;
+}
+
+/* For Desktop View */
+@media screen and (min-width: 1024px) {
+    .gfg-div {
+        background-color: #63c971;
+        color: #fff;
+    }
+}
+ 
+/* For Tablet View */
+@media screen and (min-device-width: 768px)
+    and (max-device-width: 1024px) {
+    .gfg-div {
+        width: 400px;
+        height: 400px;
+        background-color: orange;
+        color: black;
+    }
+}
+ 
+/* For Mobile Portrait View */
+@media screen and (max-device-width: 480px)
+    and (orientation: portrait) {
+    .gfg-div {
+        width: 200px;
+        height: 200px;
+        background-color: red;
+        color: #fff;
+    }
+}

```

Comment:
since this is the smallest device width, i recommend it goes to the bottom of other larger device widths